R/Simulations-class.R

```diff
@@ -359,7 +366,8 @@ DualSimulations <- function(rho_est,
     representation =
       representation(
         biomarkerFitAtDoseMostSelected = "numeric",
-        meanBiomarkerFit = "list"
+        meanBiomarkerFit = "list",
+        stop_report = "matrix"
```

Comment:
should not be needed, this is already in `SimulationsSummary` right?

Reply:
Yes, sorry for this repetitive mistake:(